nilearn/input_data/nifti_spheres_masker.py

```diff
@@ -0,0 +1,237 @@
+"""
+Transformer for computing seeds signals.
```

Comment:
Actually, I was waiting for your PR to be finalized and merged to have a common description across all maskers ;).